### PR TITLE
Made Changes Asked By GCI Task

### PR DESCRIPTION
**Pull request by Ludwant**
I added myself, according to the Google Code-in Task.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -948,7 +948,19 @@ <h1 class="person__name">Jasper Yu</h1>
                             <h1 class="person__name">Allen Chou</h1>
                         </section>
                     </div>
-                </div>
+                 </div>
+                 <div class="sympl onethird">
+                    <section class="person person--student">
+                        <div class="person__img">
+                        <div class="person__img__info"><div class="person__img__info__cont">
+
```
Comment (roonyh) [inline index.html]: please remove the unnecessary empty line

**Comment by roonyh** [comment]
Hi thanks and welcome! There some unnecessary files named ".DS_Store" in the commit. Please remove those.

**Comment by Ludwant** [comment]
Hi, sorry for the .DS_STORE files. When I added my picture and edited index.html I used "git add -a" which included those files. I fixed it now, hope everything else is fine. Thank you!

**Comment by roonyh** [comment]
Great! Thanks! Now can you please rebase your work on the updated master branch? The code in our repo has changed since you made the pull request. This makes it harder to merge. Reading this should help: https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

**Comment by Ludwant** [comment]
Alright, I rebased it. It should be good. Thanks again.

**Comment by roonyh** [comment]
Thanks! great job! just remove those unnecessary lines like "<<<<<<< HEAD" and "======="
You have done rebasing correctly except those lines are not needed and should be deleted.

And you can add a nice animation to your image with `<div class="box-team wow bounceInDown" data-wow-delay="0.3s">` Checkout other images.

**Comment by Ludwant** [comment]
I added the delay and took out the unnecessary lines. Let me know if there is something else I need to do. Thank you for your help.

File: index.html [hunk 2]
```diff
@@ -1069,6 +1072,9 @@ <h1 class="person__name">Prudhviraj Naidu</h1>
                 </section>
             </div>
         </div>
+
+                </div>
```
Comment (roonyh) [inline index.html]: This is not needed right?

File: index.html [hunk 3]
```diff
@@ -1088,6 +1094,25 @@ <h1 class="person__name">Ignacio Rodríguez</h1>
                 </section>
             </div>
         </div>
+
+                 </div>
```
Comment (roonyh) [inline index.html]: need this?

**Comment by roonyh** [comment]
Thanks :+1: ! you can check https://github.com/fossasia/fossasia.github.io/pull/110/files for changes you made. Can you please remove those unnecessary lines? and fix indentation where wrong. Thanks for your work and sorry for nit picking :smile:

**Comment by Ludwant** [comment]
Thats fine! Took out the /divs, and fixed some indentation. Sorry for those little errors. I'm guessing that with a community project, the code needs to be kept neat! Thanks again, hope everything else if okay.

**Comment by Ludwant** [comment]
Hello?

**Comment by namangoel1** [comment]
@Ludwant Merged, sorry for the delay!

**Comment by Ludwant** [comment]
Thank you so much!

**Comment by roonyh** [comment]
I am terribly sorry for not responding. I was offline for the last few days. Thanks @namangoel1 for attending to this :+1:  @Ludwant I hope the task associated is closed?

**Comment by Ludwant** [comment]
That's fine! I ran out of time on the task, so I reclaimed it, but it is not closed on the GCI website.

**Comment by roonyh** [comment]
Will close it if you could give me a link.

**Comment by Ludwant** [comment]
The link to the task is this: http://www.google-melange.com/gci/task/view/google/gci2014/5803979057397760
Thanks for all your help!

**Comment by roonyh** [comment]
Please submit a link to this pull request as the work and mark the the task as complete.

**Comment by Ludwant** [comment]
I put the link up and marked it as complete.

**Comment by Ludwant** [comment]
Thank you!

**Comment by roonyh** [comment]
You are welcome!

**Pull request merged**